eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -50,6 +50,8 @@ jobs:
       vmImage: $(OSVmImage)
 
     steps:
+      - template: ../../../common/pipelines/templates/steps/bypass-local-dns.yml
```

Comment:
`/eng/common/pipelines/templates/steps/bypass-local-dns.yml`  It is a little easier to read if you use absolute path from the root.